org.eclipse.buildship.oomph.edit/plugin.properties

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2019 the original author or authors.
```

Comment:
For some reason, I can't comment on binary files. This one is for ` org.eclipse.buildship.oomph.edit/icons/full/obj16/GradleImportTask.gif`:

❌ Where did you get this icon? All of our our images in the `org.eclipse.buildship.ui/icons/full` directories are pngs. Can we use them here as well? Also, can we make use of the `@2x` pngs?

Reply:
The standard generated class was looking for a gif, so I converted one of the pngs. I should be able to specify pngs by editing `org.eclipse.buildship.oomph.provider.GradleImportTaskItemProvider.getImage(Object)`  and excluding the method from the generation. I'm not sure if the EMFPlugin handles the high dpi images.

Reply:
Shame, we need to use low-quality images.

Reply:
I think the high dpi image handling happening in the SWT layer. It worths a shot to try.